nx/include/switch/services/psm.h

```diff
@@ -10,10 +10,11 @@
 #include "../kernel/event.h"
 
 typedef enum {
-    ChargerType_None = 0,    ///< No charger
-    ChargerType_Charger = 1, ///< Official charger or dock
-    ChargerType_Usb = 2      ///< Other USB-C chargers
-} ChargerType;
+    PsmChargerType_Unconnected = 0,  ///< No charger
+    PsmChargerType_EnoughPower = 1,  ///< Full speed
```

Comment:
`speed`? Was `power` intended?

Reply:
I meant charging at the fastest speed supported by any wattage that could be provided, but I've fixed the phrasing to be clearer.